crates/torii/migrations/20230316154230_setup.sql

```diff
@@ -3,15 +3,13 @@ CREATE TABLE indexer (
     head BIGINT NOT NULL DEFAULT 0
 );
 
-INSERT INTO indexer (head) VALUES (0);
```

Comment:
hmm do we need to remove this? it initializes the indexer

Reply:
Ah had thought that was for testing since it was in seed.sql, will add back in